frontend/src/app/modules/bim/bcf/bcf-wp-attribute-group/bcf-wp-attribute-group.component.sass

```diff
@@ -21,15 +21,20 @@ ngx-gallery ::ng-deep
     ngx-gallery-action:last-of-type .ngx-gallery-icon
       margin-right: 0
 
-  .ngx-gallery-thumbnail.ngx-gallery-clickable
-    .ngx-gallery-icon
-      font-size: 1.25rem
-      color: var(--content-link-color)
+  .ngx-gallery-thumbnail
+    background-color: white
+    &.ngx-gallery-clickable
+      .ngx-gallery-icon
+        font-size: 1.25rem
+        color: var(--content-link-color)
 
   // Disable red border on active thumbnails
   .ngx-gallery-active,
   .ngx-gallery-thumbnail
-    border: none !important
+    border: 1px solid var(--table-row-border-color) !important
```

Comment:
I am personally not a fan of using design variables just because they happen to have a matching color value. This passage here has nothing to do with the table. If that value changes, it will (unintentionally) affect the snapshot gallery. 
So, either add a new variable if you think we need the value in other places. Otherwise, it is also legitimate to  directly write the hexcode.

Reply:
Ok, I directly wrote the hexcode. I am not convinced that this is worth a fundamental variable. I am not even sure if that border will last long.